drivers/mspi/mspi_dw.h

```diff
@@ -9,24 +9,34 @@
  * It is not supposed to be included by any file other than mspi_dw.c.
  */
 
+#include <zephyr/sys/util.h>
```

Comment:
This is included by `mspi_dw.c` before `mspi_dw.h`.

Reply:
Good point-removed